### PR TITLE
feat(SDK-46): Custom Fallback Component

### PR DESCRIPTION
**Pull request by Dawntraoz**

### CODE REVIEW

**Review by alexjoverm — CHANGES_REQUESTED** [review]
@Dawntraoz the implementation is looking excellent! I just would say to add the minimal component tests to validate that:
- FallbackComponent is not enabled by default
- FallbackComponent works when enabled
- CustomFallbackComponents works when set


Same for adding the minimal docs.

**Comment by Dawntraoz** [comment]
> @Dawntraoz the implementation is looking excellent! I just would say to add the minimal component tests to validate that:
> 
> * FallbackComponent is not enabled by default
> * FallbackComponent works when enabled
> * CustomFallbackComponents works when set
> 
> Same for adding the minimal docs.

This PR is not ready for review, these are just the files I needed to double-check with you the methods used and the way I exported the `SDKoptions` to make them available to the Fallback implementation. When that's confirmed, I will push the whole PR content for review <3

File: lib/StoryblokComponent.vue [hunk 1]
```diff
@@ -1,12 +1,39 @@
...
+// Fallback component logic
+const VueSDKOptions: SbVueSDKOptions = inject("VueSDKOptions");
+
+const componentName = ref(props.blok.component);
+if (!componentFound) {
+  if (!VueSDKOptions.enableFallbackComponent) {
+    console.error(
+      `Component could not be found for blok "${props.blok.component}"! Is it defined in main.ts as "app.component("${props.blok.component}", ${props.blok.component});"?`
+    );
+    /* throw new Error(
```
Comment (Dawntraoz) [inline lib/StoryblokComponent.vue:25]: Please, @manuelschroederdev can you review if the `thrown new Error` is the best option for this or not? I think `console.error` doesn't break the JS thread and also helps the developer to know what happened, isn't it?

**Review by fgiuliani — APPROVED** [review]

File: lib/StoryblokComponent.vue (hunk 1 above)
Comment (alexjoverm) [reply lib/StoryblokComponent.vue:25]: As discussed, please better `console.error` to don't block the JS thread

**Review by alexjoverm — APPROVED** [review]

**Pull request merged**